src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -109,25 +111,55 @@ const RunPipelineDialog = (props: RunPipelineDialogProps) => {
       </Alert>
     );
   }
+  const handleClose = () => {
```

Comment:
No need for this handleClose, you can put the reset of the version picker in the useEffect above